src/main/java/fallingleaves/fallingleaves/client/FallingLeavesClient.java

```diff
@@ -1,29 +1,52 @@
 package fallingleaves.fallingleaves.client;
 
-import fallingleaves.fallingleaves.FallingLeaves;
+import fallingleaves.fallingleaves.FallingLeavesConfig;
 import fallingleaves.fallingleaves.particle.FallingLeafParticle;
 import fallingleaves.fallingleaves.particle.FallingSpruceLeafParticle;
+import me.sargunvohra.mcmods.autoconfig1u.AutoConfig;
+import me.sargunvohra.mcmods.autoconfig1u.serializer.GsonConfigSerializer;
 import net.fabricmc.api.ClientModInitializer;
 import net.fabricmc.api.EnvType;
 import net.fabricmc.api.Environment;
 import net.fabricmc.fabric.api.client.particle.v1.ParticleFactoryRegistry;
 import net.fabricmc.fabric.api.particle.v1.FabricParticleTypes;
+import net.minecraft.block.Block;
+import net.minecraft.block.Blocks;
 import net.minecraft.particle.DefaultParticleType;
 import net.minecraft.particle.ParticleType;
+import net.minecraft.util.Identifier;
 import net.minecraft.util.registry.Registry;
 
 @Environment(EnvType.CLIENT)
 public class FallingLeavesClient implements ClientModInitializer {
 
+    public static FallingLeavesConfig config;
```

Comment:
final and all caps

Reply:
Can't be final because it's initialized in `onInitializeClient()`, not in a constructor - and putting it in the constructor _might_ be a bad idea.

Reply:
Whoops, you're right.